test/serializers/non_paginated_collection_serializer_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
```

Comment:
This file should probably move to `test/non_paginated_collection_serializer_test.rb`.